### ISSUE TITLE
Avoid all services with specific (self made) cryptocurrency all they are scams!

### ISSUE THREAD

**Issue by mx5kevin** [HelloZeroNet/ZeroNet#2807]
The developers have most of these currencies. If the price goes high sell them and the users can say goodbye their money. Speculators are highly manipulating their exchange rates. There is no cover behind them, there is no guarantee that they will be exchange at any time at the fame Fiat like USD EUR etc price what their purchased. Without exception, all such services are scams, pyramid scheme! People turn their money into these services for their unique worthlessness coins and the developers, and speculators gone the money. It can cause huge loss of value within hours from 100USD within hours can go  50USD and month later 20$ than worth cents. No matter what they promise all these services are scams!

**Comment by caryoscelus** [comment]
all cryptocurrencies are self made . everyone knows 99% of crypto is worthless . please don't repeat self-evident things and treat developers with respect . this is not crypto forum , this is only for issues pertaining to zeronet (which is dead , by the way , you should use forks) . please close/remove this issue so as it doesn't clutter issue list

**Comment by mx5kevin** [comment]
> all cryptocurrencies are self made . everyone knows 99% of crypto is worthless . please don't repeat self-evident things and treat developers with respect . this is not crypto forum , this is only for issues pertaining to zeronet (which is dead , by the way , you should use forks) . please close/remove this issue so as it doesn't clutter issue list

The topic affects the program!  More and more of the crypto scammer, influencer on the ZeroNet network! And here on the GitHub topics too. Promises paid scam decentralized solutions with full of security holes and backdoors. IPFS (filecoin), Storj, Odysee LBRY, etc teams.

**Comment by purplesyringa** [comment]
> More and more of the crypto scammer

Care to share specific links?

> Promises paid scam decentralized solutions with full of security holes and backdoors

Literally ZeroNet, lol

> IPFS (filecoin)

IPFS and Filecoin are complementary, which you would know if you read the first two sentences of [the official FAQ](https://docs.filecoin.io/about-filecoin/ipfs-and-filecoin/). Filecoin is not strictly required to use IPFS, and last time I checked, no one had plans to incorporate Filecoin to ZeroNet.

> Storj, Odysee LBRY, etc teams

It's the first time I hear these titles. Where did you find them discussed on ZeroNet development boards?

**Comment by ghost** [comment]
@mx5kevin you're right. @caryoscelus is a scammer. https://github.com/zeronet-conservancy/zeronet-conservancy/pull/112 

@imachug you going to start trolling again like you did when we switched the license?

**Comment by peertopeernetworkdeveloper** [comment]
Every single fork that exist is mocked by @caryoscelus. Not only that... She also mocking and trolling on WikiPedia page of ZeroNet spamming her fork and even trying to mock others on Reddit /zeronet subreddit.

**Comment by caryoscelus** [comment]
to all bystanders or random witnesses of this unpleasant discussion : please don't let this troll cast a bad impression on the 0net people (unfortunately can't call it community , but it is what it is) at large . surely we also have trolls inside the network , but most people aren't . and we at zeronet-conservancy/riza will make every effort to build a friendly and open community while creating tools to isolate and minimize effects of destructive elements

**Comment by ghost** [comment]
@caryoscelus You trolling multiple GitHub repositories promoting your fork trying to collect money for my work and other people's work while you mocking other developers and forks on WikiPedia, Reddit and even here. You never contributed anything to ZeroNet (pretty insignificant), and what I fixed in https://github.com/zeronet-conservancy/zeronet-conservancy/pull/112 shows you don't even know how to use python. 

Your Riza scam have no future and I will do everything to make sure that everyone knows that you're a scammer who mocking other forks and developers for over a year. 

@mx5kevin is right, you're a scammer, nothing more. I will make sure the ZeroNet WikiPedia page will be cleaned up from your spam.

**Comment by ghost** [comment]
@mx5kevin Szia! 

ZeroNet is Hungarian, developed by Hungarians and will stay like that. You @caryoscelus have no chance. I'm the person who made it sure that ZeroNet has a GPv3 license and you're required by the license terms to state every single change you make in a changelog file. Since you're not doing that your repository infringes my and other people's copyrights. 

You will be reported to GitHub if I see any other comment from you mocking other forks and developers as "insecure" or providing "insecure" software. You have been spammed every single ZeroNet repository with your links and trying to collect donations for our work.
